src/faldbt/project.py

```diff
@@ -96,21 +96,29 @@ def __post_init__(self):
     def __hash__(self) -> int:
         return self.unique_id.__hash__()
 
-    def get_script_paths(self, keyword, project_dir, before) -> List[Path]:
-        return normalize_directories(project_dir, self.get_scripts(keyword, before))
+    def get_script_paths(
+        self, keyword: str, project_dir: str, before: bool
+    ) -> List[Path]:
+        if self.get_scripts(keyword, before):
```

Comment:
Since `get_scripts` now handles this case and returns `[]` itself, and since `normalize_directories` does a `map`, which works well with empty lists. Couldn't we leave this as it was? without the `if-else` check?

Reply:
yea i think we can